### PR TITLE
Allowed indexers in policies

### LINKED ISSUE #457: Cannot run when declaring a type with an indexer
https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEUCuA7AHwAEBmAAlgEMATCPAGwE8yBnDXMDMgMQggFgAUAG8hZcWVJkIwAFYxOZDAAsAliwDaqvF23UYCALpkAvAD4yeHPXoBuIQF8hQogCYSQ0YImTXrsRJePk6CDkA

```cs
public readonly struct Foo
{
    public object this[int index] => null;
}

public static class Program
{
    public static void Main()
    {
    }
}
```

> Unbreakable.AssemblyGuardException: Type System.Reflection.DefaultMemberAttribute is not allowed.

### PR DESCRIPTION
**Pull request by YohDeadfall**
Fixes ashmind/SharpLab#457.

### CODE REVIEW

File: Policy/DefaultApiPolicyFactory.cs [hunk 1]
```diff
@@ -386,14 +387,19 @@ internal class DefaultApiPolicyFactory : IDefaultApiPolicyFactory {
                 .Type(typeof(IOrderedEnumerable<>), Allowed);
         }
 
+        private static void SetupSystemReflection(NamespacePolicy globalization) {
+            globalization
```
Comment (ashmind) [inline Policy/DefaultApiPolicyFactory.cs]: This one should be `reflection`.

**Review by ashmind — COMMENTED** [review]
Thanks for your work!

Seems like policy document (.txt) didn't get updated by the build, try deleting it (from disk, not from the project) and rebuilding? It's not required for things to work, but it's useful to make sure there are no expected changes.

If after rebuild it still does not generate, that's fine, I'll update it.

File: Policy/DefaultApiPolicyFactory.cs (hunk 1 above)
Comment (YohDeadfall) [reply Policy/DefaultApiPolicyFactory.cs]: Done.

**Comment by YohDeadfall** [comment]
Yeah, deleting the file did the trick, thanks.

**Review by ashmind — APPROVED** [review]

**Comment by ashmind** [comment]
Thanks!

**Pull request merged**